backend/src/handlers/grade.ts

```diff
@@ -132,12 +132,49 @@ export const getQuestionsGrades = async (
 	}
 
 	try {
+		// Get all questions belong to the current session
+		const session = await Session.findOne({
+			where: {
+				id: sessionId,
+			},
+			attributes: ['name'],
+			include: [
+				{
+					model: Question,
+					attributes: ['id', 'title'],
+					include: [
+						{
+							model: QuestionGrade,
+							attributes: [
+								[Sequelize.fn('avg', Sequelize.col('points')), 'avgPoints'],
+								'maxPoints',
+							],
+							where: {
+								sessionId: sessionId,
```

Comment:
imagine not using the shorthand and just put `sessionId`  instead of `sessionId: sessionId` lul